### PR TITLE
Improve HANA installation description in the examples

### PR DESCRIPTION
**Pull request by arbulu89**
An improvement for the documentation where we explain how to setup the terraform configuration file to use properly the HANA installation files.

In fact, I have noticed that we use a different variable name for each of the providers....
We should unify it. Should we use `hana_inst_media` ?

FYI: @petersatsuse
#542

### CODE REVIEW

**Comment by petersatsuse** [comment]
looks much clearer as before to me

File: azure/terraform.tfvars.example [hunk 1]
```diff
@@ -92,27 +92,44 @@ public_key_location = "/path/to/your/public/ssh/key"
...
+
+# 2. Combine the `hana_inst_master` with `hana_platform_folder` variable.
+#hana_inst_master = "//YOUR_STORAGE_ACCOUNT_NAME.file.core.windows.net/sapdata/sap_inst_media"
 # Specify the path to already extracted HANA platform installation media, relative to hana_inst_master mounting point.
 # This will have preference over hana archive installation media
 #hana_platform_folder = "51053381"
 
-# Or specify the path to the HANA installation archive file in either of SAR, RAR, ZIP, EXE formats, relative to the 'hana_inst_media' mounting point
+# 3. Specify the path to the HANA installation archive file in either of SAR, RAR, ZIP, EXE formats, relative to the 'hana_inst_master' mounting point
 # For multipart RAR archives, you only need to provide the first part EXE file name.
```
Comment (MalloZup) [inline azure/terraform.tfvars.example:122]: `you only` we could remove it

File: azure/terraform.tfvars.example [hunk 2]
```diff
@@ -92,27 +92,44 @@ public_key_location = "/path/to/your/public/ssh/key"
...
-# If using hana sar archive, please also provide the compatible version of sapcar executable to extract the sar archive
+#hana_archive_file = "51053381_part1.exe"
+
+# 4. If using HANA SAR archive, please also provide the compatible version of sapcar executable to extract the SAR archive
```
Comment (MalloZup) [inline azure/terraform.tfvars.example:125]: remove `please` .
we can use imperative way for docs

**Review by MalloZup — APPROVED** [review]
looks good to me. I would remove the `you` and `please` and using imperative impersonal doc

File: libvirt/terraform.tfvars.example [hunk 3]
```diff
@@ -19,20 +19,41 @@ iprange = "192.168.XXX.Y/24"
...
+#hana_archive_file = "IMDB_SERVER.SAR"
+#hana_sapcar_exe = "SAPCAR"
+
+# For option 3 and 4, HANA installation archives are extracted to the path specified
+# at hana_extract_dir (optional, by default /sapmedia/HANA)
+#hana_extract_dir = "/sapmedia/HDBSERVER"
+
+
+# More configuration about the HANA machines
+# Set the IP addresses for the HANA machines. Let this commented to get an autogenerated addresses
```
Comment (Simranpal) [inline libvirt/terraform.tfvars.example:56]: ` Let this commented / Leave this commented` to get autogenerated addresses

**Review by Simranpal — APPROVED** [review]
Thanks @arbulu89 for the improvements. Yes, I agree we can unify and use  `hana_inst_media` for all providers

**Comment by petersatsuse** [comment]
I did yesterday additional checks as i got errors during the deployment, so i looked at the usage of the variables at all.

a) If you look at the past where the "install_master" was a DVD, it's clear that this can for sure not be the mountpoint to extract the media. 
b) Looking into the code, there is *always" hana_extract_dir used. 

The hana_extract_dir should *never* point to the _"readonly DVD"_ mountpoint of "install_master", instead there should be a normal **writeable** mountpoint and disk be used.

I have mostly on baremetal installs seen that /hana/data/<something> or /hana/backup/<something> is been used

So the result would be for example:

```
 # readonly mount point of the SAP media
  hana_inst_master = "sapfilestore.file.core.windows.net/sapbits/HANA"
 
 # HANA downloaded archive 
  hana_archive_file="51053381_part1.exe"
 
 # Extracted HANA media
  hana_extract_dir = "/hana/shared/HDBSERVER"
```

**Comment by arbulu89** [comment]
> I did yesterday additional checks as i got errors during the deployment, so i looked at the usage of the variables at all.
> 
> a) If you look at the past where the "install_master" was a DVD, it's clear that this can for sure not be the mountpoint to extract the media.
> b) Looking into the code, there is *always" hana_extract_dir used.
> 
> The hana_extract_dir should _never_ point to the _"readonly DVD"_ mountpoint of "install_master", instead there should be a normal **writeable** mountpoint and disk be used.
> 
> I have mostly on baremetal installs seen that /hana/data/ or /hana/backup/ is been used
> 
> So the result would be for example:
> 
> ```
>  # readonly mount point of the SAP media
>   hana_inst_master = "sapfilestore.file.core.windows.net/sapbits/HANA"
>  
>  # HANA downloaded archive 
>   hana_archive_file="51053381_part1.exe"
>  
>  # Extracted HANA media
>   hana_extract_dir = "/hana/shared/HDBSERVER"
> ```

Fantastic @petersatsuse ! Thank you for the research.
So, we will need to change the default `hana_extract_dir` value to something like you exposed. We will do the changes and test them in Azure.
This will probably come in a new PR though, to keep the things in order.
I will update the `terraform.tfvars.example` to show your example.

**Pull request merged**